### PR TITLE
[13.0][IMP] project_stock: Add analytic date

### PR DESCRIPTION
**Pull request by victoralmau**
The possibility of set a specific date in project or task has been added so that if it is set it will be used in analytic items linked to materials.

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT38988

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
LGTM :+1:

File: project_stock/i18n/es.po [hunk 1]
```diff
@@ -42,6 +42,12 @@ msgstr "Línea Analítica"
 msgid "Analytic Lines"
 msgstr "Líneas analíticas"
 
+#. module: project_stock
+#: model:ir.model.fields,field_description:project_stock.field_project_project__stock_analytic_date
+#: model:ir.model.fields,field_description:project_stock.field_project_task__stock_analytic_date
+msgid "Analytic date"
+msgstr "Fecha de contabilidad analítica"
```
Comment (pedrobaeza) [inline project_stock/i18n/es.po]:
```suggestion
msgstr "Fecha para analítica"
```

File: project_stock/tests/test_project_stock.py [hunk 2]
```diff
@@ -71,18 +72,35 @@ def test_project_task_analytic_lines_without_tags(self):
         self.task.write({"stage_id": self.stage_done.id})
         self.task.action_done()
         self._test_task_analytic_lines_from_task(-40)
+        self.assertEqual(
+            fields.first(self.task.stock_analytic_line_ids).date,
+            self.task.project_id.stock_analytic_date,
```
Comment (pedrobaeza) [inline project_stock/tests/test_project_stock.py]: Fixed values, please

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by victoralmau** [comment]
Changes done.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**